tutorials/bert_sentiment.py

```diff
@@ -0,0 +1,78 @@
+import onnx
```

Comment:
Does this work end to end?

Reply:
Yes, I verified it locally. It's your contribution but I don't know how I can put a history there!